src/routes/rules/RulesView.vue

```diff
@@ -490,10 +530,59 @@ export default {
 };
 </script>
 <style scoped>
+
+::-webkit-scrollbar {
+  width: 0px;
+}
+
 #logo {
   height: 20vh;
   margin: 0 auto;
 }
+.sidebar-title {
+  
+  display: flex;
+  gap: 2rem;
+}
+
+h1,
+h3,
+p {
```

Comment:
Generally it's more performant to use class based selectors. Can we put classes on the relevant tags and style them that way?